### ISSUE TITLE
ocrd workspace bulk-add --help: code formatting broken

### ISSUE THREAD

**Issue by kba** [OCR-D/core#769]
```
  \b Example:     ocrd workspace bulk-add \\             --regex
  '^.*/(?P<fileGrp>[^/]+)/page_(?P<pageid>.*)\.(?P<ext>[^\.]*)$' \\
  --file-id 'FILE_{{ fileGrp }}_{{ pageid }}' \\             --page-id
  'PHYS_{{ pageid }}' \\             --file-grp "{{ fileGrp }}" \\
  --url '{{ fileGrp }}/FILE_{{ pageid }}.{{ ext }}' \\
  path/to/files/*/*.*
```

is hard to read

**Comment by bertsky** [comment]
duplicate of #641 

(I wonder why our `\b` does not work here BTW...)

**Comment by kba** [comment]
Because the docstring has a `r` prefix, fixed in 3bfcc7ee as part of fixing #754
